tests/assets/rrd/snippets/howto/load_mcap.rrd

```diff
@@ -0,0 +1,3 @@
+version https://git-lfs.github.com/spec/v1
+oid sha256:381aaa1a292adbed4547e12be70805d67d3e3c73f358e7804bd9a052175d9861
+size 108150510
```

Comment:
I think we should avoid huge files on `main`, even on git lfs, since they will be have to be downloaded whenever the user checks out `main`, which is just slow.

Do we need these files to be so huge?

Reply:
I already cut it down to 3 seconds. I can cut it down even more

Reply:
108 MiB is _huge_ for something that everyone gets on checkout.

Reply:
i've cut it down to 18MiB, it might make sense to make this a real example that downloads the mcap file instead

Reply:
I hope I never need to download those on mobile…

Reply:
Still, better not to add to the mess